src/components/cardPreview/CardPreview.jsx

```diff
@@ -1,10 +1,13 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { Button, Card as UICard, Dropdown, Popover, Icon, Menu, Tag, Avatar, Row, Col, Modal as UIModal } from 'antd'
+import { Button, Card as UICard, Dropdown, Popover, Icon, Menu, Avatar, Row, Col, Modal as UIModal } from 'antd'
 import Modal from '../../commons/modal/Modal'
 import './style.css'
 import EditField from '../../commons/editField/EditField'
 import CardContainer from '../../containers/card/CardContainer'
+import Assignees from '../assignees/Assignees'
+// import LabelsContainer from '../../containers/labels/LabelsContainer'
+import Labels from "../labels/Labels"
 import moment from 'moment'
```

Comment:
Order the imports:
1) Libraries
2) Components / Containers
3) CSS file (or other files)